src/style.css

```diff
@@ -1 +1,130 @@
-/* add styles here */
+:root {
+  --border-color: #E1E7ED;
+}
+
+@font-face {
+  font-family: MyWebFont;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+}
+
+body {
+  margin: 0;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: AvenirBook;
+  font-weight: 300;
+}
+
+.bigSearchForm {
+  box-sizing: border-box;
+}
```

Comment:
```suggestion

```

Reply:
Please fix this line. Add this property for global tag *